project/plugins.sbt

```diff
@@ -20,6 +20,7 @@ addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.13")
 addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.0.0") // for maintenance of copyright file header
 addSbtPlugin("com.hpe.sbt" % "sbt-pull-request-validator" % "1.0.0")
+addSbtPlugin("org.scoverage" % "sbt-scoverage" % "1.6.0-M3")
```

Comment:
ok so this only does anything after explicitly enabling coverage, which we'll do at jenkins?

Reply:
Correct, the SBT tasks need to be invoked, it is not enabled by default.